### PR TITLE
[BACKLOG-39858] Preview data within each step

### PR DESCRIPTION
**Pull request by SreejaSridhar**

### CODE REVIEW

File: engine/src/main/java/org/pentaho/di/www/PreviewStepData.java [hunk 1]
```diff
@@ -0,0 +1,252 @@
+package org.pentaho.di.www;
```
Comment (hv-vamsi) [inline engine/src/main/java/org/pentaho/di/www/PreviewStepData.java:1]: Please add license text

File: engine/src/main/java/org/pentaho/di/www/PreviewStepData.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+import javax.servlet.ServletException;
+import org.pentaho.di.core.Const;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.exception.KettleStepException;
+import org.pentaho.di.core.row.RowMetaInterface;
+import org.pentaho.di.i18n.BaseMessages;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.pentaho.di.repository.*;
```
Comment (addagudi) [inline engine/src/main/java/org/pentaho/di/www/PreviewStepData.java:12]: Do not use wildcard imports

**Review by SreejaSridhar — COMMENTED** [review]
Have made changes reviewed, kindly re-review

**Review by hv-vamsi — APPROVED** [review]

**Review by sVarma1992 — APPROVED** [review]
aproving

**Pull request merged**